src/index.html

```diff
@@ -7,25 +7,24 @@
     <link rel="stylesheet" href="./style.css" />
     <script type="module" src="./main.js"></script>
     <title>Film Book</title>
-
-    <link rel="preconnect" href="https://fonts.googleapis.com" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link
       href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,700;0,900;1,100;1,300;1,700;1,900&display=swap"
-      rel="stylesheet"
-    />
-  </head>
+      rel="stylesheet">
+</head>
 
-  <body>
-    <header-component></header-component>
-    <main class="main-content gray-section">
-      <h1>Welcome to MovieCollections!</h1>
-      <h2 class="small-box">
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-        tempor incididunt ut labore et dolore magna aliqua.
-      </h2>
-      <button class="btn-primary">Go to Films!</button>
-    </main>
-    <footer-component></footer-component>
-  </body>
+<body>
+  <header-component></header-component>
+  <main class="main-content gray-section">
+    <h1>Welcome to MovieCollections!</h1>
+    <h2 class="small-box">Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+      sed do eiusmod tempor incididunt ut labore et dolore
+      magna aliqua.
+    </h2>
+    <button class="button">Go to Films!</button>
```

Comment:
nie działa mi ten url

Reply:
zobacz zmieniłaś klasę: class="btn-primary"

Reply:
sylwiagie, a week ago via PR #47   (December 13th, 2021 11:00pm) 

fix homepage and style